chargebee/src/main/java/com/chargebee/android/models/Products.kt

```diff
@@ -2,5 +2,11 @@ package com.chargebee.android.models
 
 import com.android.billingclient.api.SkuDetails
 
-data class CBProduct(val productId: String,val productTitle:String, val productPrice: String, var skuDetails: SkuDetails, var subStatus: Boolean )  {
-}
\ No newline at end of file
+data class CBProduct(
+    val productId: String,
+    val productTitle: String,
+    val productPrice: String,
+    var skuDetails: SkuDetails,
+    var subStatus: Boolean,
+    var productType: String
+)
```

Comment:
- Where is this used? 
- Should `productType` be an enum of `ProductType`?

Reply:
Yes. will change it to Enum type. This param can be used while validating the receipt on cache retry either subscriptions or one time purchases, also on product purchases.